packages/gafl-webapp-service/src/pages/terms-and-conditions/__tests__/route.spec.js

```diff
@@ -71,4 +107,9 @@ describe('terms-and-conditions get data', () => {
     const request = getMockRequest({ contactSummary: false })
     expect(() => getData(request)).rejects.toThrowRedirectTo(CONTACT_SUMMARY.uri)
   })
+
+  it.each([[true], [false]])('content returns notify content when isLicenceForYou equals %s', async licenceForYou => {
+    const { content } = await getData(getMockRequest({ isLicenceForYou: licenceForYou }))
```

Comment:
Small one again, but if you change `licenceForYou` to `isLicenceForYou`, you can do `{ isLicenceForYou }` rather than `{ isLicenceForYou: licenceForYou }`